login-workflow/src/new-architecture/components/Error/ErrorManager.tsx

```diff
@@ -0,0 +1,92 @@
+import { SxProps } from '@mui/material/styles';
+import React from 'react';
+import { BasicDialog } from '../Dialog/BasicDialog';
+import ErrorMessageBox from './ErrorMessageBox';
+
+export type AuthError = { cause: { title: string; errorMessage: string } };
+
+export type ErrorManagerProps = {
+    mode?: 'dialog' | 'message-box' | 'both' | 'none';
+    dismissible?: boolean;
+    position?: 'top' | 'bottom';
+    dismissButtonText?: string;
+    messageBoxFontColor?: string;
+    messageBoxBackgroundColor?: string;
+    errorMessage?: string;
+    onClose?: () => void;
+    messageBoxSx?: SxProps;
+    title?: string;
+    children?: React.ReactNode;
+};
```

Comment:
I would probably restructure this like:
```suggestion
export type ErrorManagerProps = {
    mode?: 'dialog' | 'message-box' | 'both' | 'none';
    dismissible?: boolean;
    onClose?: () => void;
    error: string;
    dialogConfig?: {
        title?: string;
        dismissLabel?: string;
    }
    messageBoxConfig?:{
        position?: 'top' | 'bottom';
        fontColor?: string;
        backgroundColor?: string;
        sx?: BoxProps['sx'];
    }
    children?: React.ReactNode;
};
```